app/components/custom-fields/form.tsx

```diff
@@ -131,6 +144,27 @@ export const CustomFieldForm = ({ name, helpText, required, type }: Props) => {
         </div>
       </FormRow>
 
+      <FormRow rowLabel="" className="border-b-0 pt-2">
+        <div className="flex items-center gap-3">
+          <Switch
+            name={zo.fields.active()}
+            disabled={disabled}
+            defaultChecked={true}
+            checked={isActive}
+            onCheckedChange={(checked) => setIsActive(checked)}
```

Comment:
I don't understand why we control this field? The other switch is uncontrolled and this is our preferred approach.